ckanext/pages/theme/templates_main/ckanext_pages/blog_list.html

```diff
@@ -2,7 +2,7 @@
 
 {% block page_primary_action %}
   {% if h.check_access('ckanext_pages_update', {}) %}
-    {% link_for _('Add Article'), controller='ckanext.pages.controller:PagesController', action='blog_edit', page='', class_='btn btn-primary', icon='plus-sign-alt' %}
+    {% link_for _('Add Article'), controller='ckanext.pages.controller:PagesController', action='blog_edit', page='', class_='btn btn-primary', icon='plus-square' %}
```

Comment:
!! You're right, of course. Hmm. All I can say is those places where it happens are primary actions and will manifest as buttons on standard CKANs. Hence the icon is less critical than in other places where we lose the view of the link completely.